timesketch/frontend-ng/src/views/Home.vue

```diff
@@ -146,6 +166,12 @@ export default {
     currentUser() {
       return this.$store.state.currentUser
     },
+    userSettings() {
+      return this.$store.state.settings
+    },
+    systemSettings() {
+      return this.$store.state.systemSettings
+    },
   },
   methods: {
```

Comment:
When the component is mounted it needs to update its settings to have them synced across views. Otherwise the component can't access the system settings and will display user settings only in the default setting.

```suggestion
  },
  mounted() {
    this.$store.dispatch('updateSystemSettings')
    this.$store.dispatch('updateUserSettings')
  },
  methods: {
```